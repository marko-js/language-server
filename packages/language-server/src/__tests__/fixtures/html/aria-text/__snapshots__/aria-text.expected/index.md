## Diagnostics
### Ln 1, Col 3
```marko
> 1 | p role="text"
    |   ^^^^ Type '"text"' is not assignable to type 'AttrMissing | "none" | "search" | "list" | "link" | "dialog" | "grid" | "listbox" | "menu" | "tree" | "alert" | "alertdialog" | "application" | "article" | "banner" | ... 55 more ... | "treeitem"'.
  2 |   -- body text
  3 |   button
```

### Ln 1, Col 1
```marko
> 1 | p role="text"
    | ^ Fix any of the following:
  Element has focusable descendants
  2 |   -- body text
  3 |   button
```

