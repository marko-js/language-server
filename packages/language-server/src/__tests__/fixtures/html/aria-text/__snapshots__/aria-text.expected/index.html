<p data-marko-node-id="0" role="text">body text<button data-marko-node-id="1"></button></p>